modules/lib.lua

```diff
@@ -294,6 +294,7 @@ else
 
     ---@class LibDrawText3DParams : LibDrawTextParams
     ---@field coords vector3
+    ---@field drawRect? boolean default: `true`
```

Comment:
Maybe make it `disableDrawRect` instead?

Reply:
yeah maybe that'd be better lol